compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -288,6 +342,21 @@ mkDataDecl env thisModule occName tyVars cons = do
                 }
         }
 
+-- | Make a binding of the form "x = x". If a qualified name is passed,
+-- we turn the left-hand side into the unqualified form of that name (LHS
+-- must always be unqualified), and the right-hand side remains qualified.
+mkTrivialBind :: Located RdrName -> HsBind GhcPs
```

Comment:
Maybe `mkStubBind` would be a better name?

Reply:
I'll change it in the next PR, thanks!